doc/ReleaseNotes.md

```diff
@@ -13,29 +13,171 @@ r151034 release repository: https://pkg.omniosce.org/r151034/core
 
 ## Upgrade Notes
 
+It is only possible to upgrade to this release from r151030 or r151032. If
+starting from an earlier release, upgrade in stages following the guide at
+https://omniosce.org/upgrade
+
 ## New features since r151032
 
 ### System Features
 
+* OmniOS now supports running an NFS server in a zone. NFS sharing can be
+  enabled by setting the `sharenfs` property on a dataset just as in the
+  global zone.
+
+* While it was previously possible to create an SMB share within a zone via
+  the `sharemgr` command, it can now be done more easily via the `sharesmb`
+  dataset property, as in the global zone.
+
```

Comment:
With the caveat that all of the other things you had to do in the global zone (AD, etc. etc. if needed) ALSO need to be done in-zone.